lib/screens/about_page.dart

```diff
@@ -15,8 +22,50 @@ class AboutScreen extends StatelessWidget {
     );
   }
 
+  String getVersionString(Version? newVersion) {
+    var v = version.toString();
+
+    if (newVersion != null) {
+      v += " (neue Version " + newVersion.toString() + " verfügbar)";
+    }
+
+    return v;
+  }
+
+  Future<VersionAndUrl?> getVersionAndUrl() async {
+    var v = await getNewVersion();
+    var url = await newVersionUrl();
+
+    return new VersionAndUrl(v, url);
+  }
+
+  Future<Version?> getNewVersion() {
+    return gitHub.repositories.listTags(repositorySlug).asyncMap((element) {
+      try {
+        return Version.parse(element.name.replaceFirst(RegExp(r'v|V'), ''));

```

Comment:
Why create a new RegExp per element, instead of declaring it as a member in the class?